sdk/storage/azblob/README.md

```diff
@@ -0,0 +1,362 @@
+# Azure Storage Blob SDK for Go
+
+## Introduction
+
+The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud
+storage. This SDK replaces the previously previewed [azblob package](https://github.com/azure/azure-storage-blob-go) in the separate repository.
+
+## Getting Started
+
+The Azure Blob SDK can access an Azure Storage account.
+
+### Prerequisites
+
+* Go versions 1.16 or higher. If you don't already have it, install [the Go distribution](https://golang.org/dl/)
+* You must have an [Azure storage account][azure_storage_account]
+
+#### Create account
+
+* To create a new Storage account, you can use [Azure Portal][azure_portal_create_account], [Azure PowerShell][azure_powershell_create_account], or [Azure CLI][azure_cli_create_account]:
+
+### Install the package
+* Install the Azure blob storage for Go with `go get`:
+  ```bash
+  go get github.com/Azure/azure-sdk-for-go/sdk/storage/azblob
+  ```
+  
+#### Create the client
+
+`azblob` allows you to interact with three types of resources :-
+
+* [Azure storage accounts][azure_storage_account].
+* [Containers](https://azure.microsoft.com/en-in/overview/what-is-a-container/#overview) within those storage accounts.
+* [Blobs](https://azure.microsoft.com/en-in/services/storage/blobs/#overview) (block blobs/ page blobs/ append blobs) within those containers.
+
+Interaction with these resources starts with an instance of a [client](#clients). 
+To create a client object, you will need the account's blob service endpoint URL and a credential that allows you to access the account. 
+The `endpoint` can be found on the page for your storage account in the [Azure Portal][azure_portal_account_url] under the "Access Keys" section or by running the following Azure CLI command:
+
+```bash
+# Get the table service URL for the account
+az storage account show -n mystorageaccount -g MyResourceGroup --query "primaryEndpoints.table"
+```
+
+Once you have the account URL, it can be used to create the service client:
+```golang
+cred, err := azblob.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
+serviceClient, err := azblob.NewServiceClient("https://<myAccountName>.blob.core.windows.net/", cred, nil)
+handle(err)
+```
+
+For more information about blob service URL's and how to configure custom domain names for Azure Storage check out the [official documentation][azure_portal_account_url]
+
+#### Types of credentials
+
+The clients support different forms of authentication. Storage accounts can use a Shared Key Credential, Connection
+String, or a Shared Access Signature Token for authentication
+in [`azidentity`](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity)
+like `azidentity.NewDefaultAzureCredential()`.
```

Comment:
This second sentence doesn't make sense. Does storage support AAD? if so, your list is not formed correctly or it might be leftover copy paste

Reply:
Looks like it does based on your samples, this second sentence needs to be fixed